automation/taskcluster/lib/tasks.py

```diff
@@ -161,6 +163,50 @@ def craft_test_task(self, variant):
             },
         )
 
+   
+    def craft_ui_tests_task(self):
+        artifacts = {
+            "public": {
+                "type": "directory",
+                "path": "/build/fenix/results",
+                "expires": taskcluster.stringDate(taskcluster.fromNow(DEFAULT_EXPIRES_IN))
+            }
+        }
+
+        env_vars = {
+            "GOOGLE_PROJECT": "moz-fenix",
```

Comment:
Can you add me to this project? I can't see it on the console yet. :)

Reply:
I made you an owner awhile back and it says your invite is pending acceptance.  I don't see a way to 'resend' invite, so I've deleted and re-added you as an owner.  Hopefully you'll see an invite in your inbox now

Reply:
Thank you. I got it! :)